scripts/php-mise.sh

```
@@ -0,0 +1,164 @@
+done
+
+echo "Installing PHP extensions"
+
+pecl_bin update-channels 1> /dev/null
+
+ext_dir=`php_bin -r "echo ini_get('extension_dir').PHP_EOL;"`
+exts=( $( echo ${!pecl_exts[@]} | tr ' ' $'\n' | sort ) )
+
+for ext in ${exts[@]}; do
+    if [[ ! -f "$ext_dir/$ext.so" ]]; then
+        install_ext pecl $ext ${pecl_exts[$ext]}; continue
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)